src/code/z_demo.c

```diff
@@ -1,99 +1,1634 @@
+        func_801A2D54(cmd->sequence - 1);
+    }
+}
+
+// Command 0x9C: Fade Background Music or Fanfare over duration
+void Cutscene_Command_FadeSequence(GlobalContext* globalCtx, CutsceneContext* csCtx, CsCmdMusicFade* cmd) {
+    if ((csCtx->frames == cmd->startFrame) && (csCtx->frames < cmd->endFrame)) {
+        u8 frameCount = cmd->endFrame - cmd->startFrame;
+
+        if (cmd->type == 2) {
+            Audio_QueueSeqCmd((frameCount << 0x10) | 0x110000FF);
+        } else {
+            Audio_QueueSeqCmd((frameCount << 0x10) | NA_BGM_STOP);
+        }
+    }
+}
+
+// Command 0x12E: Play Ambience music
+void Cutscene_Command_PlayAmbienceSequence(GlobalContext* globalCtx, CutsceneContext* csCtx, CsCmdBase* cmd) {
+    if (csCtx->frames == cmd->startFrame) {
+        // Audio_PlayNatureAmbienceSequence
+        // nightSeqIndex is be natureAmbienceId
+        func_801A4A28(globalCtx->soundCtx.nightSeqIndex);
+    }
+}
+
+// Command 0x130:
+void func_800EAD48(GlobalContext* globalCtx, CutsceneContext* csCtx, CsCmdBase* cmd) {
+    if (csCtx->frames == cmd->startFrame) {
+        // Audio_SetSfxReverbIndexExceptOcarinaBank
+        func_801A4428(2);
+    }
+}
+
+// Command 0x131:
+void func_800EAD7C(GlobalContext* globalCtx, CutsceneContext* csCtx, CsCmdBase* cmd) {
+    if (csCtx->frames == cmd->startFrame) {
+        // Audio_SetSfxReverbIndexExceptOcarinaBank
+        func_801A4428(1);
+    }
+}
+
+#ifdef NON_MATCHING
+// needs in-function static bss
+// audio related
+void func_800EADB0(GlobalContext* globalCtx, CutsceneContext* csCtx, CsCmdBase* cmd) {
+    static u16 activeSequence;
+    u8 dayMinusOne;
+
+    if (csCtx->frames == cmd->startFrame) {
+        dayMinusOne = (gSaveContext.day - 1);
+        if (dayMinusOne >= 3) {
+            dayMinusOne = 0;
+        }
+
+        switch (cmd->base) {
+            case 1:
+                // func_801A246C(SEQ_PLAYER_BGM_MAIN, TYPE_1);
+                func_801A246C(0, 1);
+                break;
+
+            case 2:
+                // func_801A246C(SEQ_PLAYER_BGM_MAIN, TYPE_0);
+                func_801A246C(0, 0);
+                break;
+
+            case 3:
+                // func_801A246C(SEQ_PLAYER_BGM_MAIN, TYPE_2);
+                func_801A246C(0, 2);
+                break;
+
+            case 4:
+                // func_801A246C(SEQ_PLAYER_NATURE, TYPE_1);
+                func_801A246C(4, 1);
+                break;
+
+            case 5:
+                // func_801A246C(SEQ_PLAYER_NATURE, TYPE_0);
+                func_801A246C(4, 0);
+                break;
+
+            case 6:
+                // func_801A246C(SEQ_PLAYER_NATURE, TYPE_2);
+                func_801A246C(4, 2);
+                break;
+
+            case 7:
+                // Audio_GetActiveSequence
+                activeSequence = func_801A8A50(0);
+                break;
+
+            case 8:
+                if (activeSequence != 0xFFFF) {
```

Comment:
```suggestion
                if (seqId != NA_BGM_DISABLED) {
```
I think `seqId` is fine here